src/main/java/core/basesyntax/FigureBase.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class FigureBase implements FigureArea, FigureDrawable {
+    protected String color;
+
+    public FigureBase(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public abstract double getArea();
+
+    @Override
+    public abstract void getInfo();
```

Comment:
The method name `getInfo` does not seem to be conceptually similar to `getArea` or drawing a figure. If `getInfo` is intended to print information about the figure, it might be better to have a separate interface for printing or describing the figure, as the `draw()` and `getArea()` methods are conceptually different from `getInfo`.